app/_layout.tsx

```diff
@@ -0,0 +1,11 @@
+import { Stack } from "expo-router";
```

Comment:
Update on this comment:

I was able to test the app without error with `npx expo start --clear`
I'd still change the import because that is how it is documented